app/controllers/info_controller.rb

```diff
@@ -16,101 +16,24 @@ def show
     @needs = @content["links"]["meets_user_needs"]
 
     begin
-      metadata = GOVUK::Client::MetadataAPI.new.info(@slug).to_h
+      statistics = PerformanceData::Statistics.new(@content, @slug)
 
-      document_type = @content["document_type"] || @content["format"]
-
-      part_urls = get_part_urls(@content, @slug)
-      @is_multipart = is_multipart(part_urls, document_type)
-      calculated_metrics = metrics_from(metadata.fetch("performance"), part_urls, @is_multipart)
+      @is_multipart = statistics.is_multipart
+      calculated_metrics = statistics.metrics
       @lead_metrics = calculated_metrics[:lead_metrics]
       @per_page_metrics = calculated_metrics[:per_page_metrics]
     rescue StandardError => e
```

Comment:
I think that makes sense, since there's still a significant amount of indirection that I feel could be simplified. I've created a card for this: https://trello.com/c/oVwtZ40Z/165-improve-use-of-statistics-in-info-frontend